src/reset-password/service.ts

```diff
@@ -48,10 +48,10 @@ export async function sendResetPasswordEmail(user: User) {
 }
 
 /**
- * This function will create a unique token then store it in the databse
+ * This function will create a unique token then store it in the database
  */
 export async function createToken(user: User): Promise<string> {
-  const token = crypto.randomBytes(32).toString('base64').replace(/+/g, '-').replace(/\//g, '_').replace(/=+$/, '');
+  const token = crypto.randomBytes(32).toString('base64').replace(/'+'/g, '-').replace(/\//g, '_').replace(/=+$/, '');
```

Comment:
What's going on here?

Reply:
This was throwing an error
`SyntaxError: Invalid regular expression: /+/: Nothing to repeat`

Reply:
Well you fixed the error but the regex is wrong